packages/dto/src/tables/users/user.rs

```diff
@@ -39,9 +39,12 @@ pub struct User {
     #[api_model(version = v0.1, indexed, unique)]
     pub username: String,
 
-    #[api_model(one_to_many = followers, foreign_key = user_id)]
+    #[api_model(one_to_many = followers, foreign_key = following_id)]
     #[serde(default)]
     pub followers: Vec<Follower>,
+    #[api_model(one_to_many = followers, foreign_key = follower_id)]
+    #[serde(default)]
+    pub following: Vec<Follower>,
```

Comment:
```suggestion
    #[api_model(many_to_many = followers, foreign_table_name = users, foreign_reference_key = following_id, foreign_primary_key = follower_id)]
    #[serde(default)]
    pub followers: Vec<User>
    #[api_model(many_to_many = followers, foreign_table_name = users, foreign_reference_key = follower_id, foreign_primary_key = following_id)]
    #[serde(default)]
    pub followings: Vec<User>,
```

This may be correct and optimized relationship definition.